eng/common/scripts/Package-Properties.ps1

```diff
@@ -129,19 +129,26 @@ function Get-PrPkgProperties([string]$InputDiffJson) {
             if ($shouldInclude) {
                 $packagesWithChanges += $pkg
 
-                if ($pkg.DependentPackages) {
-                    $dependentPackagesForInclusion += $pkg.DependentPackages
+                if ($pkg.AdditionalValidationPackages) {
+                    $dependentPackagesForInclusion += $pkg.AdditionalValidationPackages
                 }
             }
         }
     }
 
     foreach ($addition in $dependentPackagesForInclusion) {
```

Comment:
minor given is just a local variable but we might want to make `dependentPackagesForInclusion` match the rename as well.